src/mgmt/common/kube_obj.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020 The Authors.
+
+# Authors: Sherif Abdelwahab <@zasherif>
```

Comment:
For new files, make sure Authors are correct

Reply:
Fixed!